.github/workflows/podman_tests.yaml

```diff
@@ -0,0 +1,36 @@
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    branches:
+      - main
+
+
+jobs:
+  podman-tests:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - run: sudo apt-get -y update
+      - run: sudo apt-get install -y pkg-config libsystemd-dev libdbus-glib-1-dev libelf-dev libseccomp-dev libgpgme-dev
+      - run: make build 
+      - run: sudo cp youki /usr/local/bin
+      - name: Clone podman repository
+        uses: actions/checkout@v2
+        with:
+          repository: containers/podman
+      - uses: actions/setup-go@v2
+        with:
+          go-version: '1.18'
+      - name: Build podman
+        run: make binaries 
+      - name: Install tools
+        run: make install.tools
+      - name: Install bats
+        run: sudo ./hack/install_bats.sh
+      - name: Run podman test
+        run: sudo OCI_RUNTIME=/usr/local/bin/youki ./hack/bats 2>&1 | tee build.log 
+      - name: Adding Summary
+        run: |
+          echo "Total tests: 360 Failed tests: $(cat build.log | grep " ok " | wc -l)" >> $GITHUB_STEP_SUMMARY
```

Comment:
The expression is currently not evaluated. The test just prints
`"Total tests: 360 Failed tests: $(cat build.log | grep " ok " | wc -l)" >> $GITHUB_STEP_SUMMARY`

Reply:
I didn't wanted to add a RED status to the CI.